src/metabase/models/params/custom_values.clj

```diff
@@ -54,30 +54,44 @@
   Maybe we should lower it for the sake of displaying a parameter dropdown."
   1000)
 
+(defn- last-stage-number
```

Comment:
unclear if it's worth requiring `mbql.u` for a single small function, but fyi there is a similar func in `mbql.u/legacy-last-stage-number` (currently not public)